src/olympia/versions/models.py

```diff
@@ -131,6 +131,10 @@ class Version(OnChangeMixin, ModelBase):
     git_hash = models.CharField(max_length=40, blank=True)
     source_git_hash = models.CharField(max_length=40, blank=True)
 
+    recommendation_status = models.PositiveSmallIntegerField(
+        choices=amo.RECOMMENDATION_VERSION_STATUS_CHOICES.items(),
+        default=amo.STATUS_NULL)
+
```

Comment:
Do we really need that extra field ? We're going to have to update `status` on the `DiscoveryItem` anyway when a version is approved, right ?

Reply:
We need to track which specific version (or file) has been approved - otherwise the developer could delete the approved versions and we'd be recommending a version that was unreviewed.  I'm 99% certain this was something we agreed was necessary in the previous meeting :)

I'm not sure if we need 3 states on `DiscoveryItem` though - we have NULL, AWAITING_REVIEW and APPROVED but NULL and some other non-NULL state are probably enough because if the `Addon.current_version` isn't `STATUS_APPROVED` then it's not offered as recommended.

Reply:
You're totally right, we did agree on that, I even pointed out in the PRD that it'd be stored per version/file. Carry on :) 

Maybe a simple boolean would work for both fields ?

Reply:
Replaced both `DiscoveryItem` and `Version` field with `BooleanField`s https://github.com/mozilla/addons-server/pull/11275/commits/9e44673d0f3352b206ac6e188f2ea2c1d1a10dc3